src/main/webapp/app/exercises/shared/manage/exercise-create-buttons.component.ts

```diff
@@ -36,4 +41,19 @@ export class ExerciseCreateButtonsComponent implements OnInit {
             this.router.navigate(['course-management', this.course.id, this.exerciseType + '-exercises', result.id, 'import']);
         });
     }
+
+    getExerciseTypeIcon(exerciseType: ExerciseType) {
```

Comment:
This code already exists in `ExamExerciseImportComponent`. Please unify it in a common util or service class